firmware/library/L1_Drivers/onboardled.hpp

```diff
@@ -7,14 +7,96 @@
 #pragma once
 
 #include <cstdint>
+#include "L1_Drivers/gpio.hpp"
+#include "L2_Utilities/macros.hpp"
 
 class OnBoardLedInterface {
  public:
-           virtual void Initialize()                                 = 0;
-           virtual void LedOn(uint8_t led_number)                    = 0;
-           virtual void LedOff(uint8_t led_number)                   = 0;
-           virtual void LedSet(uint8_t led_number, bool on)          = 0;
-           virtual void LedToggle(uint8_t led_number)                = 0;
-           virtual void LedSetAll(uint8_t value)                     = 0;
-           virtual uint8_t LedGetValues(void)                        = 0;
+  enum LightState : bool
+  {
+  kOn = true,
+  kOff = false
+  };
+           virtual void Initialize()                                      = 0;
+           virtual void On(uint8_t led_number)                            = 0;
+           virtual void Off(uint8_t led_number)                           = 0;
+           virtual void Set(uint8_t led_number,
+              OnBoardLedInterface::LightState state)                      = 0;
```

Comment:
You don't have to prefix this with `OnBoardLedInterface::` since you are within scope of that namespace.

Reply:
This has been fixed.